sunpy/io/tests/test_filetools.py

```diff
@@ -21,101 +21,101 @@ def test_read_file_network_fits():
     # but it is mocked to prevent network access
     url = "https://hesperia.gsfc.nasa.gov/rhessi_extras/imagecube_fits/2015/12/20/20151220_2228_2248/hsi_imagecube_clean_20151220_2228_13tx3e.fits"
     with patch("astropy.io.fits.file.download_file", return_value=TEST_AIA_IMAGE) as mock:
-        hdulist = sunpy.io.read_file(url)
+        hdulist = read_file(url)
         assert mock.call_args[0][0] == url
     assert isinstance(hdulist, list)
     assert len(hdulist) == 1
     assert len(hdulist[0]) == 2
     assert isinstance(hdulist[0][0], np.ndarray)
-    assert isinstance(hdulist[0][1], sunpy.io.header.FileHeader)
+    assert isinstance(hdulist[0][1], sunpy.io._header.FileHeader)
```

Comment:
Won't this fail since you've removed `import sunpy.io`?

Reply:
I still import it no?!

Reply:
Why does that still work?!

It surely won't get it from `import sunpy` s?!

Reply:
I imported it for my sanity.